SleepNumberController_App.groovy

```diff
@@ -767,162 +772,164 @@ def processBedData(responseData) {
   def outletData = [:]
   def underbedLightData = [:]
 
-  def deviceTypes = getBedDeviceTypes()
+  List deviceTypes = getBedDeviceTypes()
 
   for (def device : getBedDevices()) {
-    if (!outletData.get(device.getState().bedId)) {
-      outletData[device.getState().bedId] = [:]
-      underbedLightData[device.getState().bedId] = [:]
+    String bedId=device.getState().bedId.toString()
```

Comment:
nit: space before and after =

Reply:
If you like adjust to to your style conventions